docs/guides/how-to-contribute-leaderboards-or-test-reports.md

```diff
@@ -0,0 +1,54 @@
+# How to contribute leaderboards or test reports
+
+
+This document helps you prepare environment for developing code for Ianvs.
```

Comment:
Not environment though. This paragraph serves as the very beginning which need to polish carefully.